chains/mainnet/Carbon.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  chainID: 'carbon-1',
+  lcd: 'https://api.carbon-network',

```

Comment:
I think this URL is wrong

Reply:
sorry was a typo, it's "https://api.carbon.network" , have edited it